script/Ciel/Day 10/10_01_CIEL10_5_2.txt

```diff
@@ -342,7 +343,7 @@ It must already be far too late for me.
 -- Page 401, Offset 28174.
 -- 「―――、―――……！！！！」
 // lint-off:LintEmDashes
-"―――, ―――!!!!"
+"――――――!!!!"
```

Comment:
We should probably try and tag a lot of these if we can

These lines that are just nothingness we've generally tried to actually have some letters in (these are gagged Shiki)

>Mmmmgh!

Reply:
![image](https://user-images.githubusercontent.com/90473809/236602246-b44ee145-5a43-41ad-9bd7-68165402f87d.png)
These don't look fun to find and edit. I'll still try and replace what I find while proofreading though.